warehouse/tuf/README.md

```diff
@@ -0,0 +1,242 @@
+# General TUF Warehouse implementation Notes
```

Comment:
This should probably all move into our [docs](https://github.com/pypi/warehouse/tree/main/docs).